packages/alchemy/src/react/hooks/useLogout.ts

```diff
@@ -0,0 +1,46 @@
+"use client";
+
+import { useMutation, type UseMutateFunction } from "@tanstack/react-query";
+import { useAlchemyAccountContext } from "../context.js";
+import type { BaseHookMutationArgs } from "../types.js";
+import { useSigner } from "./useSigner.js";
+
+export type UseLogoutData = void;
+
+export type UseLogoutParams = void;
```

Comment:
```suggestion
export type UseLogoutParams = {};
```

Reply:
or 
```suggestion
export type Empty = {};

export type UseLogoutParams = Empty;
```

Reply:
let's keep void, since react query interprets thats as not needing any params